From e76cc53810923b2de9af43134a4456279ed1a0f6 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Thu, 13 Sep 2018 10:08:48 +0200
Subject: [PATCH 03/11] common: Allow appdomain to get vendor_camera_prop

 * Some apps rely on this and won't work properly if they
   can't read this property. See ~ Snapchat.

Change-Id: I737e28f038941e6d18b96ff512f29dbe0ad7b7e4
---
 common/private/app.te | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 common/private/app.te

diff --git a/common/private/app.te b/common/private/app.te
new file mode 100644
index 0000000..9f418bd
--- /dev/null
+++ b/common/private/app.te
@@ -0,0 +1,2 @@
+# Allow appdomain to get vendor_camera_prop
+get_prop(appdomain, vendor_camera_prop)
-- 
2.17.1

